src/state/slices/stakingDataSlice/stakingDataSlice.ts

```diff
@@ -2,11 +2,17 @@ import { createSlice } from '@reduxjs/toolkit'
 import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
 import { CAIP10, caip10 } from '@shapeshiftoss/caip'
 import { ChainAdapter } from '@shapeshiftoss/chain-adapters'
+import { CosmosSdkBaseAdapter } from '@shapeshiftoss/chain-adapters/dist/cosmossdk/CosmosSdkBaseAdapter'
 import { chainAdapters, ChainTypes } from '@shapeshiftoss/types'
 import { getChainAdapters } from 'context/PluginProvider/PluginProvider'
 
+export type PubKey = string
 type AllStakingDataArgs = { accountSpecifier: CAIP10 }
 
+type AllValidatorDataArgs = { chain: ChainTypes }
```

Comment:
can we use chainId here over ChainTypes - we're trying to move away from this

Reply:
done